core/src/main/scala/kafka/server/ControllerConfigurationValidator.scala

```diff
@@ -47,7 +52,17 @@ class ControllerConfigurationValidator extends ConfigurationValidator {
         }
         LogConfig.validate(properties)
       case BROKER =>
-        // TODO: add broker configuration validation
+        if (resource.name().nonEmpty) {
```

Comment:
Seems like we probably have other places where we validate broker IDs from strings. Is there a utility method we can use here?

Reply:
Hmm... I'm not aware of any. `ZkAdminManager` just uses `Integer.valueOf`, as does `TopicCommand` and `KafkaConfig`.